.github/workflows/deploy-the-button.yaml

```diff
@@ -143,6 +143,30 @@ jobs:
           source contracts/env/${{ env.CONTRACTS_ENVFILE }} && echo -n "$NODE" > env_NODE.txt
           aws s3 cp env_NODE.txt s3://${{ secrets.CI_S3BUCKET_NAME }}/${{ env.S3BUCKET_PATH }}/env_NODE.txt
 
+      - name: Rename metadata and addresses JSON files to artifact upload
+        shell: bash
+        run: |
+          cp contracts/addresses.json addresses.featurenet.json
```

Comment:
you can use you can use `env.CONTRACTS_ENVFILE` here

Reply:
changed